cargo-dist/src/init.rs

```diff
@@ -1428,3 +1428,25 @@ fn apply_optional_mac_pkg(
         table.remove(key);
     }
 }
+
+/// Similar to [`apply_optional_value`][] but specialized to `MinimumGlibcVersion`, since we're not able to work with structs dynamically
```

Comment:
can you paste what the init experience is for this?

Reply:
It's used like this at line 1310:

```rust
    apply_optional_min_glibc_version(
        table,
        "minimum-glibc-version",
        "# The minimum glibc version supported by the package (overrides auto-detection)\n",
        minimum_glibc_version.as_ref(),
    );
```

Reply:
ah my bad- i meant what an end user would see

Reply:
Since I had to change the field names for some of the v1 config to make this work, I changed all of it to avoid weird mismatches between "minimum" and "min" being scattered all over the place.

Reply:
Oh! There's two variants.

First, as its own table:

```
[dist.min-glibc-version]
"*" = "2.17"
i686-unknown-linux-gnu = "2.18"
```

Second, this is the equivalent when put directly under `[dist]`. I don't recommend it unless you're only overriding one value, but TOML allows it:

```
[dist]
# ...
min-glibc-version."*" = "2.17"
min-glibc-version.i686-unknown-linux-gnu = "2.18"
```

Reply:
haha yeah i saw this in the code and it's so wild looking haha